tasks/83_missing-term/missing-term.js

```diff
@@ -0,0 +1,8 @@
+export function findMissing(arr) {
+  const mean = (arr[arr.length - 1] - arr[0])/arr.length;
```

Comment:
Introducing some vars here could increase readability. Something like `lastElement`, `firstElement`.
Also adding some comments for `for` (pun intended) would be useful.

Reply:
Fixed readability and added comments.